src/SmartComponents/PatchManager/PatchManagerCard.js

```diff
@@ -22,13 +22,18 @@ import messages from '../../Messages';
 import { useIntl } from 'react-intl';
 
 /**
- * Operating systems card for showing the ratio of operating systems used.
+ * Card for showing the systems and ratios of current advisories.
  */
-const PatchManagerCard = ({ systems, systemsStatus, fetchSystems, fetchSecurity, securityStatus,
-    security, bugs, fetchBugs, bugsStatus, enhancements, fetchEnhancements, enhancementsStatus,
-    selectedTags, workloads, SID }) => {
+const PatchManagerCard = ({
+    systems, systemsStatus, fetchSystems,
+    advisories, advisoriesStatus, fetchAdvisories,
+    selectedTags, workloads, SID
+}) => {
     const intl = useIntl();
-    const isLoaded = [systemsStatus, securityStatus, bugsStatus, enhancementsStatus].every(item => item === 'fulfilled');
+    const isLoaded = [systemsStatus, advisoriesStatus].every(item => item === 'fulfilled');
+    const security = advisories.security;
```

Comment:
As @gkarat mentioned, we need to handle the initial render when `advisories` is undefined. We can achieve this with object destruction: 
```suggestion
    const { security, bugfix: bugs, enhancement: enhancements } = advisories || {};
```